backend/app/routes/folders.py

```
@@ -417,14 +417,23 @@ def disable_ai_tagging(request: UpdateAITaggingRequest):
     """Delete multiple folders by their IDs."""
     try:
         if not request.folder_ids:
             raise ValueError("No folder IDs provided")
 
         deleted_count = db_delete_folders_batch(request.folder_ids)
 
+        # Best-effort: re-curate to prune memories orphaned by the delete; a failed submit shouldn't mask a delete that already succeeded.
+        executor: ProcessPoolExecutor = app_state.executor
+        try:
+            executor.submit(_curate_memories, "folder_delete")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)